smurfs/src/components/SmurfList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import { getSmurfs } from '../actions'
+
+import Smurf from './Smurf'
+
+function SmurfList(props) {
+    // API GET on mount
+    const dispatch = useDispatch()
+
+    React.useEffect(()=>{
+        dispatch(getSmurfs())
+    },[dispatch])
+
+    // get smurfs
+    const { smurfs } = useSelector(({smurfs})=>({smurfs}))

```

Comment:
If you're using `useSelector()` is probably is a better idea to use this syntax, just so it's easier to understand for other developers
```
const smurfs = useSelector(state => state.smurfs)
```